wordpress-mobile/WordPress-Android
# Add Colors screen for Design System

Part of issue https://github.com/wordpress-mobile/WordPress-Android/issues/19802

This PR adds the `Colors` screen to the design system section in the `Me` tab.

-----

## To Test:
1. Navigate to the `Me` tab. If you're on a debug build, you should see the `Design System` row.
2. Click `Foundation` -> `Colors`. 
3. You should see the new `Colors` screen and the title of the screen should update accordingly.
4. Go back to the `Me` tab and navigate to `App Settings` -> `Appearance` -> change the mode to dark or light mode. 
5. Go back to the `Colors` screen and you should see the colors updated (i.e Primary Foreground will be black in light mode and will be white in dark mode).

| Dark mode | Light mode |
|-----------|------------|
|   <img width="306" alt="Screenshot 2024-02-22 at 4 03 04 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/ae2e8c8e-feb7-4ebb-89e7-9cd975767c0a"> |  <img width="308" alt="Screenshot 2024-02-22 at 4 03 14 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/347440d9-958d-4709-ba5a-fca2b114f3c7">     |


-----

## Regression Notes

1. Potential unintended areas of impact

    - None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - None

7. What automated tests I added (or what prevented me from doing so)

    - None

-----

## PR Submission Checklist:


-----

## Testing Checklist:
